plugins/SSH.koplugin/main.lua

```diff
@@ -136,16 +137,31 @@ function SSH:addToMainMenu(menu_items)
         sub_item_table = {
             {
                 text = _("Start SSH server"),
-                callback = function() return self:start() end,
+                keep_menu_open = true,
+                callback = function(touchmenu_instance)
+                    self:start()
+                    -- sleeping might not be needed, but it gives the feeling
+                    -- something has been done and feedback is accurate
```

Comment:
lol, I suppose

Reply:
Would it make more sense to do a quick little info popup (like when changing font size and such)?

Reply:
I think that this menu items update is fine and enough as a feedback, you'll see :) Just like our checkboxes are fine: you tap, the menu gets updated with what is the current setting.
(The additional top of screen notifications feels to me a bit extra stuff, and should not be needed now that we have pretty checkboxes that give the adequate feedback - and they are often one of the culprits in the papercut/tearing glitches - contributing to many parallel refreshes busy-ness.)

I'm not trying to find out what is best for starting/stoping SSH (2 menu items Start/Stop with only one enabled as it is now, or a checkbox like most other stuff, with or without an additional popup notification). 
I just found out that if we keep menu open, we need to update menu items (nothing was needed when the menu was closed). So I did that for what currently exists: the Start and Stop menu items. This `sleep 1` just gives (me) a better feeling that the feedback is accurate :)

Reply:
Sometimes an action like changing a font, while reading a book, can take some time.  A popup that informs the user that the action is underway, is a good idea in those cases.

Reply:
> I think that this menu items update is fine and enough as a feedback, you'll see

I mean I don't care for the sleep tbh. It's more of an *if*...